lib/solidus_subscriptions/subscription_generator.rb

```diff
@@ -29,6 +29,7 @@ def activate(subscription_line_items)
         billing_address: order.bill_address,
         payment_source: order.payments.valid.last&.payment_source,
         payment_method: order.payments.valid.last&.payment_method,
+        currency: order.currency,
```

Comment:
It's better to reflect this change also in the [spec](https://github.com/solidusio-contrib/solidus_subscriptions/blob/master/spec/lib/solidus_subscriptions/subscription_generator_spec.rb#L20-L28).